### PR TITLE
Support mobile display

### PR DESCRIPTION
**Pull request by pavloburchak**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/organisms/Sider/Sider.styled.tsx [hunk 1]
```diff
@@ -39,14 +39,14 @@ export const StyledSider = styled(Layout.Sider)<{$isFullScreenLogOutput?: boolea
...
-export const StyledSiderChildContainer = styled.div`
+export const StyledSiderChildContainer = styled.div<{$width: number}>`
   position: fixed;
   display: flex;
   flex-direction: column;
   justify-content: space-between;
 
   padding-top: 30px;
-  width: 100px;
+  width: ${props => props.$width}px;
```
Comment (rangoo94) [inline src/components/organisms/Sider/Sider.styled.tsx:49]: I would consider `zoom` instead, applied via media queries. I don't think there would be any edge cases, but what do you think?

```css
.ant-layout-sider {
  zoom: 0.6;
}

.ant-layout-sider-children > div > div {
  zoom: 1.35;
}
```

**Review by rangoo94 — COMMENTED** [review]
The most important thing is that we should:
* avoid depending on JS for displaying on different devices (always when it's possible), and
* avoid setting direct dimensions of elements (unless it's part of their geometry, or would be much harder otherwise)
This way, we may depend on browser mechanisms, ensuring that the behavior is smooth; mobile devices are worse than desktop, so it's even more important for them.

Besides code review, I've checked how it works in real, and works nicely 👍 

| Screenshot | Description |
|-|-|
| Zrzut ekranu 2023-08-9 o 08 37 52 | **Priority:** <kbd>High</kbd><br>May be worth to delete help for creating executors on mobile, or make it smaller |
| Zrzut ekranu 2023-08-9 o 08 37 23 | **Priority:** <kbd>Medium</kbd><br>Modal for creating the test should be constrained to maximum width |
| Zrzut ekranu 2023-08-9 o 08 37 02 | **Priority:** <kbd>High</kbd><br>May be worth deleting the creating tests help on mobile |
| Zrzut ekranu 2023-08-9 o 08 36 41 | **Priority:** <kbd>Low</kbd><br>It should wrap (or at least truncate?) for some minimum size of these items |
| Zrzut ekranu 2023-08-9 o 08 34 11 | **Priority:** <kbd>Low</kbd><br>I'm thinking about this one - would it be possible to move the graph to right side of labels? If that would require some bigger effort - ignore it 🙂 |
| Zrzut ekranu 2023-08-9 o 08 38 06 | **Priority:** <kbd>Low</kbd><br>Should "Condition" / "Actions" steps list wrap? |
| Zrzut ekranu 2023-08-9 o 08 40 30 | **Priority:** <kbd>High</kbd><br>The tabs are strangely positioned - the tab content should be scrollable, but tabs rather always visible |
| Zrzut ekranu 2023-08-9 o 08 41 33 | **Priority:** <kbd>Very Low</kbd><br>Completely minor - maybe it's worth to change minimum size for official executors, to make them 2 in a row on mobile? |

File: src/components/organisms/EntityList/EntityListFilters/EntityListFilters.tsx [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
 import LabelsFilter from './LabelsFilter';
 import StatusFilter from './StatusFilter';
 import TextSearchFilter from './TextSearchFilter';
 
 const EntityListFilters: FC<FilterProps> = props => {
   const {isFiltersDisabled, ...rest} = props;
 
+  const isMobile = useIsMobile();
+
+  const filtersWidth = isMobile ? 'inherit' : '296px';
```
[the remark this answers is not in the document]
Comment (pavloburchak) [reply src/components/organisms/EntityList/EntityListFilters/EntityListFilters.tsx:17]: I think I do want it, but ok, I can change it

File: src/components/molecules/EntityGrid/EntityGridItemPure.tsx [hunk 3]
```diff
@@ -70,6 +72,9 @@ const EntityGridItemPure = forwardRef<HTMLDivElement, EntityGridItemPureProps>((
     [onAbort, item]
   );
 
+  const {width} = useWindowSize();
+  const isDisplayExecutionTime = width > size.mobileL;
```
[the remark this answers is not in the document]
Comment (pavloburchak) [reply src/components/molecules/EntityGrid/EntityGridItemPure.tsx:76]: you mean with display none?

File: src/components/organisms/EntityList/EntityListFilters/EntityListFilters.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (rangoo94) [reply src/components/organisms/EntityList/EntityListFilters/EntityListFilters.tsx:17]: `initial` would make the container use its default behavior, while `inherit` will use the same value as the parent container - not the computed one, but the relative one.

As an example, if you'll have HTML ([**codesandbox**](https://codesandbox.io/s/summer-snow-9mctlf?file=/index.html)):

```html
<style>
div {width: 30%;height: 20px;background: rgba(0, 0, 0, .1);}
.inherit {width: inherit;}
.initial {width: initial;}
</style>

<div><div class="inherit"><div class="inherit"></div></div></div>
<br />
<div><div class="initial"><div class="initial"></div></div></div>
```

Therefore, I believe that you did rather want `initial`, but I may be wrong 🙂

File: src/components/molecules/EntityGrid/EntityGridItemPure.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (rangoo94) [reply src/components/molecules/EntityGrid/EntityGridItemPure.tsx:76]: Yes, exactly 🙂

File: src/components/organisms/EntityList/EntityListFilters/EntityListFilters.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pavloburchak) [reply src/components/organisms/EntityList/EntityListFilters/EntityListFilters.tsx:17]: Thanks, will update it

File: src/components/organisms/Sider/Sider.styled.tsx (hunk 1 above)
Comment (pavloburchak) [reply src/components/organisms/Sider/Sider.styled.tsx:49]: Haven't seen this approach later, this one is not looking perfectly in mobile

**Comment by rangoo94** [comment]
> for me these tabs display ok, could you provide more details?

Zrzut ekranu 2023-08-10 o 08 50 20

The tabs may not be visible, as the whole container (along with tabs) becomes scrollable on smaller heights (= mobile). The tabs should be rather always visible though, while only content (i.e. Log Output) should scroll.

**Review by rangoo94 — COMMENTED** [review]
| Screenshot | Description |
|-|-|
| Zrzut ekranu 2023-08-11 o 08 19 45 | Displaying the message - the buttons should wrap, and optionally we could make it smaller |
| Zrzut ekranu 2023-08-11 o 08 23 23 | Full screen Log Output doesn't work |
| Zrzut ekranu 2023-08-11 o 08 24 33 | Optional: it would be great if the full screen icon would go down. Also, probably the "AI Insights" tooltip could have fixed position, but I can do that in a separate PR. |

File: src/components/organisms/Sider/Sider.tsx [hunk 4]
```diff
@@ -199,9 +203,11 @@ const Sider: React.FC = () => {
     });
   }, []);
 
+  const siderWidth = isMobile ? 60 : 100;
+
   return (
     <StyledSider width={100} data-cy="navigation-sider" $isFullScreenLogOutput={isFullScreenLogOutput}>
-      <StyledSiderChildContainer>
+      <StyledSiderChildContainer $width={siderWidth}>
```
Comment (rangoo94) [inline src/components/organisms/Sider/Sider.tsx:210]: `$width` is unused

File: src/styles/globalStyles.ts [hunk 5]
```diff
@@ -479,29 +480,14 @@ export const GlobalStyle = createGlobalStyle`
...
-};
+  @media ${maxDevice.tablet} {
+    .ant-layout-sider {
+      zoom: 0.6;
+    }
+
+    .ant-layout-sider-children > div > div {
+      zoom: 1.35;
+    }
+  }
```
Comment (rangoo94) [inline src/styles/globalStyles.ts:492]: This could be part of `Sider` implementation

**Review by rangoo94 — APPROVED** [review]

**Pull request merged**